openedx/core/djangoapps/credentials/utils.py

```diff
@@ -0,0 +1,88 @@
+"""Helper functions for working with Credentials."""
+from __future__ import unicode_literals
+import logging
+
+from edx_rest_api_client.client import EdxRestApiClient
+
+from openedx.core.djangoapps.credentials.models import CredentialsApiConfig
+from openedx.core.djangoapps.programs.utils import get_programs_for_credentials
+from openedx.core.lib.token_utils import get_id_token
+
+
+log = logging.getLogger(__name__)
+
+
+def get_user_credentials(user):
```

Comment:
I have used this name as here we are getting credentials specific to the users not all. Using the name `get_credentials` will give the impression that we are getting all the credentials which is not the case here. If you think `get_credentials` makes more sense than I can change it